zerver/lib/external_accounts.py

```diff
@@ -10,12 +10,16 @@
 
 DEFAULT_EXTERNAL_ACCOUNTS = {
     "twitter": {
-        "text": "Twitter",
-        "url_pattern": "https://twitter.com/%(username)s"
+        "text": "Twitter",  # Realm admin sees in field options
```

Comment:
Let's move these comments explaining what the values do to a block comment at the top of `DEFAULT_EXTERNAL_ACCOUNTS`